osu.Game.Tests/Visual/Playlists/TestSceneAddPlaylistToCollectionButton.cs

```diff
@@ -53,7 +52,11 @@ public void SetUpSteps()
         {
             AddStep("clear realm", () => Realm.Realm.Write(() => Realm.Realm.RemoveAll<BeatmapCollection>()));
 
-            AddStep("clear notifications", () => notificationOverlay.AllNotifications.Empty());
```

Comment:
i can understand the other sharpcompress import hijink but how did ***THIS*** happen :exploding_head:

Reply:
Yeah it's pretty amusing isn't it. I glossed over when reviewing the tests.

I don't know why such a helper method even exists (check its implementation for even more WTF)

Reply:
> check its implementation for even more WTF

:see_no_evil: :speak_no_evil: